.github/workflows/pre-commit.yml

```
@@ -2,10 +2,14 @@ name: Pre-commit checks
 
 on:
   push:
-    branches: [ "**" ]
+    branches: [ master ]
```

YAMLlint (1.38.0)

[error] too many spaces inside brackets

(brackets)